schema.json

```diff
@@ -0,0 +1,514 @@
+        },
+        "Expression": {
+          "type": "string"
+        }
+      },
+      "additionalProperties": false,
+      "required": []
+    },
+    "fields": {
+      "type": "array",
+      "items": {
+        "type": "object",
+        "properties": {
+          "Name": {
+            "type": "string"
+          },
+          "Type": {
+            "type": "string"
+          },
+          "Initializer": {
+            "type": "string"
+          },
+          "Modifiers": {
+            "type": "number"
+          },
+          "Attributes": {
+            "$ref": "#/$defs/attributes"
+          },
+          "DocumentationComments": {
+            "$ref": "#/$defs/documentationComments"
+          }
+        },
+        "additionalProperties": false,
+        "required": [
+          "Name",
+          "Type"
+        ]
+      }
+    },
+    "documentationComments": {
+      "type": "object",
+      "properties": {
+        "Example": {
+          "type": "string"
+        },
+        "Exceptions": {
+          "type": "object"
+        },
+        "Params": {
+          "type": "object"
+        },
+        "Permissions": {
+          "type": "object"
+        },
+        "Remarks": {
+          "type": "string"
+        },
+        "Returns": {
+          "type": "string"
+        },
+        "SeeAlsos": {
+          "type": "object"
+        },
+        "Summary": {
+          "type": "string"
+        },
+        "TypeParams": {
+          "type": "object"
+        }
+      },
+      "additionalProperties": false,
+      "required": []
+    },
+    "attributes": {
+      "type": "array",
+      "items": {
+        "type": "object",
+        "properties": {
+          "Name": {
+            "type": "string"
+          },
+          "Type": {
+            "type": "string"
+          },
+          "Arguments": {
+            "type": "array",
+            "items": {
+              "type": "object",
+              "properties": {
+                "Name": {
+                  "type": "string"
+                },
+                "Type": {
+                  "type": "string"
+                },
+                "Value": {
+                  "type": "string"
+                }
+              },
+              "additionalProperties": false,
+              "required": [
+                "Name",
+                "Type",
+                "Value"
+              ]
+            }
+          }
+        },
+        "additionalProperties": false,
+        "required": [
+          "Name",
+          "Type"
+        ]
+      }
+    },
+    "arguments": {
+      "type": "array",
+      "items": {
+        "type": "object",
+        "properties": {
+          "Type": {
+            "type": "string"
+          },
+          "Text": {
+            "type": "string"
+          }
+        },
+        "additionalProperties": false,
+        "required": [
+          "Type",
```

Comment:
I would agree with `Type` being required, but it looks like the [Pitstop example JSON](https://github.com/eNeRGy164/LivingDocumentation.Workshop/blob/main/2.living-documentation/solutions/21.LivingDocumentation/pitstop.analyzed.json) doesn't always have this, on line 8117. Do we say that's a bug in the example JSON?

Reply:
Ah, good spot. It could be a bug, but unless we plan on fixing it, I don't think we can set Type as required...